herd/AArch64ParseTest.ml

```diff
@@ -61,12 +61,10 @@ module Make(Conf:RunTest.Config)(ModelConfig:MemCat.Config) = struct
             X.X.run
       end in
 
-(* START NOTWWW *)
-      if Conf.variant Variant.ASL then
+    if Conf.variant Variant.ASL then begin
```

Comment:
Why `begin`/`end` ?

Reply:
I guess some debug printing was inserted and later removed. It does not really harm.